apps/coscrad-frontend/src/components/resources/spatial-features/leaflet/coscrad-leaflet-map.tsx

```diff
@@ -1,4 +1,5 @@
 import { LatLngExpression, Map } from 'leaflet';
+import 'leaflet/dist/leaflet.css';
```

Comment:
Moved this out of the main map index presenter into the leaflet specific component.  I still can't see a way to remove this particular css import.

Reply:
Ok. Let's leave it be for now.